gke-deploy/core/crd/crd.go

```diff
@@ -0,0 +1,42 @@
+// Package crd contains logic related to CRDs.
+package crd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/cloud-builders/gke-deploy/services"
+)
+
+const (
+	applicationCRDName       = "customresourcedefinition.apiextensions.k8s.io/applications.app.k8s.io"
+	applicationCRDInstallURI = "https://raw.githubusercontent.com/kubernetes-sigs/application/master/config/crd/bases/app.k8s.io_applications.yaml"
+)
+
+// EnsureInstallApplicationCRD ensures the installation of the Application CRD in the current context's cluster.
+func EnsureInstallApplicationCRD(ctx context.Context, ks services.KubectlService) error {
+	installed, err := crdIsInstalled(ctx, applicationCRDName, ks)
+	if err != nil {
+		return err
+	}
+	if installed {
+		return nil
```

Comment:
Should we have a log here if the crd has been installed?

Reply:
I didn't feel like it would be useful. I want the installation of the CRD to be transparent